src/main/java/com/efficio/etl/service/impl/ETLServiceImpl.java

```diff
@@ -935,11 +935,7 @@ public StudyDetails readStudyDetails(final Sheet sheet) {
 			ETLServiceImpl.STUDY_DETAILS_VALUE_COLUMN_INDEX);
 		final String studyType = this.getCellStringValue(sheet, ETLServiceImpl.STUDY_TYPE_ROW_INDEX - rowAdjustMent,
 			ETLServiceImpl.STUDY_DETAILS_VALUE_COLUMN_INDEX);
-		StudyTypeDto studyTypeValue = this.studyDataManager.getStudyTypeByName(studyType);
-		if (studyTypeValue == null) {
-			// TODO we need to change what to do when Study Type is not in the file
-			studyTypeValue = this.studyDataManager.getStudyTypeByName(StudyTypeDto.NURSERY_NAME);
-		}
+		final StudyTypeDto studyTypeValue = this.studyDataManager.getStudyTypeByName((StudyTypeDto.TRIAL_LABEL.equals(studyType) ? StudyTypeDto.TRIAL_NAME : StudyTypeDto.NURSERY_NAME));
```

Comment:
Created technical backlog ticket [IBP-3050](https://ibplatform.atlassian.net/browse/IBP-3050) to check whether some code in `ETLServiceImpl `can be removed, seems to be redundant with code in `WorkbookParser `class in Middleware